### DISCUSSION TITLE
Strelka alerts not populating fields

### DISCUSSION THREAD

**Discussion by wompRS** [Security-Onion-Solutions/securityonion#5959] [Unsupported Versions] [answered]
I have Strelka rules enabled which are currently triggering an alert on a possible Equation Group infection. I am 99% sure this is a false positive, however, the functionality I am expecting with ETOpen rules is not occurring with Strelka rules.

Essentially, none of the strelka information is populating in the alerts or hunt dashboard. When you expand the detection each individual field is there, but you can't search by anything except the timestamp or rule name to correlate alerts.

Picture for reference.

What occurs:
![image](https://user-images.githubusercontent.com/8816079/138144615-592a44a1-e099-457b-b6b3-5028f031f87d.png)

What is expected: 
![image](https://user-images.githubusercontent.com/8816079/138145274-434ba120-4802-4f6a-bbd9-545c5f2964a0.png)

**Comment by dougburks** [comment, marked as answer]
Are you asking why the Strelka log does not have anything under the network columns in the top screenshot? Strelka just sees the file itself and not the network stream that transferred the file. Therefore, there is no IP address or port information.
